### ISSUE TITLE
The pre-trained model provided is not the same size as your own model

### ISSUE THREAD

**Issue by shakey-cuimiao** [ultralytics/yolov5#1273]
The model best.pt trained by yolov5s is 58M, but the pre-trained model yolov5s.pt is 15M. Has the pre-trained model been cropped

**Comment by glenn-jocher** [comment]
@shakey-cuimiao models while training incorporate FP32 gradients. Completed training model sizes are here.

<img width="1127" alt="97808084-edfcb100-1c64-11eb-83eb-ffed43a0859f" src="https://user-images.githubusercontent.com/26833433/97982332-ead40300-1dd3-11eb-8c7e-0b49c4eaa3bb.png">
